src/script_interface/LocalContext.hpp

```diff
@@ -39,10 +39,11 @@ namespace ScriptInterface {
  */
 class LocalContext : public Context {
   Utils::Factory<ObjectHandle> m_factory;
+  bool m_is_head_node;
 
 public:
-  explicit LocalContext(Utils::Factory<ObjectHandle> factory)
-      : m_factory(std::move(factory)) {}
+  explicit LocalContext(Utils::Factory<ObjectHandle> factory, int mpi_rank)
```

Comment:
I guess the `explicit` was only necessary because of [Core Guideline C46](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#c46-by-default-declare-single-argument-constructors-explicit)

Reply:
I wasn't sure. In any case, I have more changes pending on the script interface. If it is not needed anymore, I'll remove the `explicit` in one of the next PRs.